### PR TITLE
Updates content of Services page

### PR DESCRIPTION
**Pull request by locks**
Fixes https://github.com/simplabs/simplabs.github.io/issues/808.

### CODE REVIEW

File: src/ui/components/PageServices/stylesheet.css [hunk 1]
```diff
@@ -3,8 +3,32 @@
 @block layout from "../../styles/blocks/layout.block.css";
 @block offset from "../../styles/blocks/offset.block.css";
 @block typography from "../../styles/blocks/typography.block.css";
+@block button from "../../styles/blocks/button.block.css";
 
 :scope {
   block-name: PageServices;
   composes: 'layout';
 }
+
```
Comment (ghislaineguerin) [inline src/ui/components/PageServices/stylesheet.css]: @locks I believe we already have classes for multi-column layouts? Why aren't we reusing those?

File: src/ui/components/PageServices/stylesheet.css [hunk 2]
```diff
@@ -3,8 +3,32 @@
...
+  .split-principles-leading {
+    grid-column: 1/1;
+  }
+  .split-principles-trailing {
+    grid-column: 3/3;
+  }
+}
+
+.text-align-left {
+  text-align: left;
```
Comment (ghislaineguerin) [inline src/ui/components/PageServices/stylesheet.css]: @locks Why are we adding a global helper on a page specific stylesheet?

File: src/ui/components/PageServices/template.hbs [hunk 3]
```diff
@@ -7,49 +7,126 @@
...
+
+  <div class="layout.full offset.after-12">
+    <h3 class="typography.h3">
+      <small class="typography.small">Digital Products</small>
+      Transform bold ideas into shipped products
+    </h3>
+    <p class="typography.lead">
+      Our team of experts delivers everything from ideation to design and engineering. We can turn any idea sketched on the back of a napkin into a final, shipped product. 
+    </p>
+    <a class="button" href="/contact/" data-internal>
```
Comment (ghislaineguerin) [inline src/ui/components/PageServices/template.hbs:76]: This button needs some margin to separate from previous paragraph.

File: src/ui/components/PageServices/stylesheet.css (hunk 1 above)
Comment (ghislaineguerin) [reply src/ui/components/PageServices/stylesheet.css]: @locks Can we also not use class names that reference content? As content changes these will become obsolete.

**Comment by ghislaineguerin** [comment]
image
This text is too small, can we try using the lead class?
All else looks good.

**Comment by ghislaineguerin** [comment]
@locks there's a test failing. Please fix it so that we can merge this.

**Comment by locks** [comment]
I'm not sure why the tests were failing since they passed locally, but I also pushed the title case changes we discussed.

**Review by ghislaineguerin — APPROVED** [review]

**Pull request merged**